crates/store/blobs/src/lib.rs

```diff
@@ -235,7 +240,10 @@ impl Blob {
             return Ok(None);
         };
 
-        #[allow(clippy::semicolon_if_nothing_returned)]
+        #[expect(
+            clippy::semicolon_if_nothing_returned,
+            reason = "False positive; not possible with macro"
+        )]
```

Comment:
is this complaining about code generated by the macro?

Reply:
Yup - as per the reason added 🙂